General/views.py

```diff
@@ -99,4 +101,61 @@ def get(self, request):
         else:
             self.context['other_associations'] = Association.objects.all()
 
-        return render(request, self.template, self.context)
\ No newline at end of file
+        return render(request, self.template, self.context)
+
+
+class EmailTemplateView(View):
```

Comment:
I suspected that that was what the view was for, I'd still like to have that also written down in the docstring so that it is clear without needing to ask (my original comment).